### DISCUSSION TITLE
r-4.0.4 on arm64 gets killed

### DISCUSSION THREAD

**Discussion by lindig** [Homebrew/discussions#907] [Everyday usage] [answered]
```
==> Pouring r-4.0.4.arm64_big_sur.bottle.tar.gz
🍺  /opt/homebrew/Cellar/r/4.0.4: 2,144 files, 62.3MB
lindig@mini ~ % R
zsh: killed     R
```

I've already re-installed R and looked around the internet to find other reports of the problem but could not. I would be grateful if others could verify if this is happening to them as well and for suggestions how to debug this, if this is not a universal problem.

**Comment by lindig** [comment, marked as answer]
Just found the solution based on https://github.com/Homebrew/discussions/discussions/439:

```
lindig@mini ~ % brew reinstall gettext pcre2

```

**Reply by carlocab** [reply]
That should work. So you don't need to try each dependency manually, you can also do
```
brew reinstall $(brew deps r)
```
